### DISCUSSION TITLE
CVE-2021-44832﻿ and CVE-2021-45105

### DISCUSSION THREAD

**Discussion by kedras** [lunasec-io/lunasec#405] [Feature Request]
Hi,
would you pls confirm whether the 1.5.2 scanner version can also detect CVE-2021-44832﻿ and CVE-2021-45105? Was not able to find it anywhere myself.

Thank you very much

**Comment by freeqaz** [comment]
We wrote up a blog post about this here: https://www.lunasec.io/docs/blog/log4j-hype-train

These vulnerabilities are far less impactful and we're not showing them in the scanner currently because, at least in the case of CVE-2021-45105, if you're vulnerable to that CVE then you have much bigger problems on your hand!

We very much don't want to increase the "hype" around these vulnerabilities because their impact is far, far less than the original Log4Shell vulnerability.

**Reply by kedras** [reply]
Fair enough. Thank you.
